lib/sample_rate_converter/Kconfig

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SAMPLE_RATE_CONVERTER
+	bool "Sample Rate conversion library"
+	select CMSIS_DSP
+	select CMSIS_DSP_FILTERING
+	select TIMING_FUNCTIONS
+	help
+	  Enable the sample rate conversion library. The library uses CMSIS DSP filters to
+	  preserve quality during the conversion. 16k, 24k and 48k are supported.
```

Comment:
Is 24 supported?

Reply:
It is - but I realized this text is a bit confusing as it stands. I've specified that it's referring to kHz, so hopefully that makes it clearer what the numbers are.